requirements.txt

```diff
@@ -2,3 +2,4 @@ cryptography==2.7
 PyMySQL==0.9.3
 pytest==4.6.3
 google-api-python-client==1.7.9
+google-auth==1.6.3
```

Comment:
This is a requirement of `google-api-python-client` as well - do you need to list is separately or would it be better to just use it as a dependency of `google-api-python-client`?

Reply:
I don't think I need to list this separately.